tests/p2p/dumb_peer.py

```diff
@@ -0,0 +1,23 @@
+from p2p import eth
+from p2p.peer import (
+    ETHPeer,
+)
+
+from p2p import protocol
+
+
+class DumbPeer(ETHPeer):
+    _supported_sub_protocols = [eth.ETHProtocol]
+    sub_proto = None  # type: eth.ETHProtocol
+    head_td = None  # type: int
+    head_hash = None  # type: bytes
```

Comment:
The reason I suggest you inherit from `ETHPeer` is just so that you don't have to duplicate those attributes here. You should be able to remove them now